Assets/Scripts/UI/DialogBox/DialogBoxManager.cs

```diff
@@ -39,42 +39,66 @@ public void Awake()
 
         GameObject tempGoObj;
 
-        tempGoObj = CreateDialogGO("DB_SaveFile", "Save File");
-        dialogBoxSaveGame = tempGoObj.GetComponent<DialogBoxSaveGame>();
-        DialogBoxes["Save File"] = dialogBoxSaveGame;
+        if (SceneController.Instance.IsAtMainScene())
+        {
+            tempGoObj = CreateDialogGO("DB_SaveFile", "Save File");
+            dialogBoxSaveGame = tempGoObj.GetComponent<DialogBoxSaveGame>();
+
+            tempGoObj = CreateDialogGO("DB_Options", "Options");
+            dialogBoxOptions = tempGoObj.GetComponent<DialogBoxOptions>();
+
+            tempGoObj = CreateDialogGO("DB_Trade", "Trade");
+            dialogBoxTrade = tempGoObj.GetComponent<DialogBoxTrade>();
+
+            tempGoObj = CreateDialogGO("DB_JobList", "Job List");
+            dialogBoxJobList = tempGoObj.GetComponent<DialogBoxJobList>();
+
+            tempGoObj = CreateDialogGO("DB_Quests", "Quests");
+            dialogBoxQuests = tempGoObj.GetComponent<DialogBoxQuests>();
+            AddQuestList();
 
-        tempGoObj = CreateDialogGO("DB_LoadFile", "Load File");
-        dialogBoxLoadGame = tempGoObj.GetComponent<DialogBoxLoadGame>();
-        DialogBoxes["Load File"] = dialogBoxSaveGame;
+            AddMainMenuItems();
+        }else
+        {
+            tempGoObj = CreateDialogGO("DB_SaveFile", "Save File");
```

Comment:
There is a problem with your merge conflict here. You don't load the good GameObject on the good scenes.